server/utils/auth.js

```diff
@@ -32,6 +32,18 @@ module.exports = {
     // next();
     return req;
   },
+  authenticate: async function(auth) {
```

Comment:
This method will authenticate a user and return the user data. Only use this function when the mutation/query requires it. See resolver.js for an example.